src/Message.ts

```diff
@@ -243,19 +239,21 @@ export class DecodedMessage {
   contentType: ContentTypeId
   content: any // eslint-disable-line @typescript-eslint/no-explicit-any
   error?: Error
+  contentBytes: Uint8Array
 
   constructor({
     id,
     messageVersion,
     senderAddress,
     recipientAddress,
     conversation,
+    contentBytes,
```

Comment:
probably dumb question, but why do we need both `contentBytes` and `content: any` members?

Reply:
Oh I think I get it, usually you'll have both members but when deserializing off disk we start with `contentBytes` and then use `decodeContent` to get `content`.

Reply:
Bingo.

I thought about making `contentBytes` a private property, but decided it was harmless to expose and might be useful for constructing test cases.